airframe-sql/src/main/scala/wvlet/airframe/sql/analyzer/TypeResolver.scala

```diff
@@ -336,6 +335,8 @@ object TypeResolver extends LogSupport {
       case _ =>
         List(expr)
     }
+    trace(s"findMatchInInputAttributes: ${expr}, inputAttributes: ${inputAttributes} -> ${results}")
```

Comment:
Moved from the head of the method to here in order to include lookup results in the log message.